persistence/rest/src/main/java/org/infinispan/persistence/rest/configuration/RestStoreConfiguration.java

```diff
@@ -33,22 +33,20 @@
 
    public static final AttributeDefinition<String> KEY2STRING_MAPPER = AttributeDefinition.builder("key2StringMapper", WrappedByteArrayOrPrimitiveMapper.class.getName()).immutable().xmlName("key-to-string-mapper").build();
    public static final AttributeDefinition<String> METADATA_HELPER = AttributeDefinition.builder("metadataHelper", EmbeddedMetadataHelper.class.getName()).immutable().build();
-   public static final AttributeDefinition<String> PATH = AttributeDefinition.builder("path", "/").immutable().build();
-   public static final AttributeDefinition<Boolean> APPEND_CACHE_NAME_TO_PATH = AttributeDefinition.builder("appendCacheNameToPath", false).immutable().build();
+   public static final AttributeDefinition<String> CACHE_NAME = AttributeDefinition.builder("cacheName", "default").immutable().build();
```

Comment:
IMO the default cache name should be the name of the cache from the `InitializationContext`, so that one could define a cache template with a RestStore and use it for multiple caches.

Reply:
ok